mail_channel_notify_tag/views/mail_templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <template id="mail_channel_message_repost" name="Mail Channel Message Repost">
+        <div style="width: 100%; margin: 0 auto;">
+            <div style="margin: 5px 0;">
+                New message on:
```

Comment:
This header is redundant, when we get a message we know we have a "New message on:"

Please remove,

Reply:
Additionally, I would preserve the original format:

```python
{{SenderName}} - {{Time}} on {{RelatedDocument}}
```

example:

<img width="1006" alt="image" src="https://github.com/Vauxoo/addons-vauxoo/assets/3836433/a2d996d4-15f7-47eb-8537-52549e5d004d">

Reply:
Is this a message in channel?

Reply:
this is an example in v14

<img width="1616" alt="image" src="https://github.com/Vauxoo/addons-vauxoo/assets/3836433/9468c658-58ad-4342-84cf-01daf75b06d9">

Reply:
The problem is those messages had channel_ids, but we lost that reference, but I will try to replicate that behavior. That is why we sent a message directly to the channel in the message_post.

Reply:
Hi, as we discussed, I think this is not possible because the messages are not sent from the records to the channels anymore, actually the suggested format is still native [here](https://github.com/odoo/odoo/blob/2c39f6cb2e984df459c649569e5ecd3bfeb01169/addons/mail/static/src/components/message/message.xml#L117), but we are sending the message in the channels directly, so originThread is the same channel and it gives no more information, but I made and improvement to the template.

![Screenshot from 2023-09-19 20-11-27](https://github.com/Vauxoo/addons-vauxoo/assets/90422721/a2ce579b-c2ca-43bb-9dec-5f0d4f9caf9c)